supertokens_python/recipe/session/session_functions.py

```diff
@@ -322,6 +325,7 @@ async def refresh_session(
     refresh_token: str,
     anti_csrf_token: Union[str, None],
     disable_anti_csrf: bool,
+    user_context: Optional[Dict[str, Any]] = None,
```

Comment:
this is an internal function, so do not set it to None. Otherwise you may miss out on passing this.

Reply:
Same goes for other functions in this file where you are passing in user_context